toolkit/tools/pkg/imagecustomizerlib/verityutils.go

```diff
@@ -135,3 +137,12 @@ func roundUpToPowerOf2(n uint32) uint32 {
 	}
 	return res
 }
+
+func getVerityNames(verity []verityDeviceMetadata) []string {
+	names := make([]string, len(verity))
+	for i, v := range verity {
+		names[i] = v.name
+	}
+	verityNames := sliceutils.RemoveDuplicatesFromSlice(names)
```

Comment:
The names should be unique. So, `RemoveDuplicatesFromSlice` is unnecessary.

Reply:
Oh I see, I noticed in the api docs that the name value can only ever be `root` or `usr`. So I thought if there are multiple usr verity entries in the config, then the result of this list would be `[usr, usr, usr]`. If there is more than one verity in the config, will it just be one `usr` and one `root` max?

Reply:
anyways, updated to remove that function call